packages/core/src/i18n/translations.ts

```diff
@@ -13,164 +13,163 @@
+    SMS: 'SMS',
+    Submit: 'Soumettre',
+    'User already exists': "L'utilisateur existe déjà",
+    'User does not exist': "L'utilisateur n'existe pas",
+    Username: "Nom d'utilisateur",
     'Username cannot be empty': "Le nom d'utilisateur doit être renseigné",
+    'Username/client id combination not found.': "L'utilisateur n'existe pas",
+    Verify: 'Vérifier',
+    'Verify Contact': 'Vérifier le contact',
   },
 
   es: {
-    'Sign in to your account': 'Iniciar sesíon',
-    'Loading...': 'Cargando...',
-    'Confirmation Code': 'Codigo de confirmación',
-    'Sign In': 'Iniciar sesíon',
-    'Sign Up': 'Regístrase',
-    'Sign Out': 'Desconectar',
-    'Forgot Password': 'Se te olvidó la contraseña?',
-    Username: 'Nombre de usuario',
-    Password: 'Contraseña',
+    'Account recovery requires verified contact information':
+      'La recuperación de la cuenta requiere información de contacto verificada',
+    'Back to Sign In': 'Iniciar sesión',
     'Change Password': 'Cambia la contraseña',
-    'New Password': 'Nueva contraseña',
-    Email: 'Email',
-    'Phone Number': 'Número de teléfono',
-    'Resend Code': 'Mandar codigo otra vez',
-    'Create a new account': 'Crear una cuenta nueva',
+    Code: 'Código',
+    Confirm: 'Confirmar',
     'Confirm a Code': 'Confirmar un código',
     'Confirm Sign In': 'Confirmar inicio de sesión',
     'Confirm Sign Up': 'Confirmar Registración',
-    'Back to Sign In': 'Iniciar sesión',
-    'Send Code': 'Enviar código',
-    Confirm: 'Confirmar',
-    'Resend a Code': 'Reenviar un código',
-    Submit: 'Enviar',
-    Skip: 'Omitir',
-    Verify: 'Verificar',
-    'Verify Contact': 'Verificar contacto',
-    Code: 'Código',
-    'Account recovery requires verified contact information':
-      'La recuperación de la cuenta requiere información de contacto verificada',
-    'Username cannot be empty': 'El campo de usuario no puede estar vacido',
-    'User does not exist': 'El usuario no existe',
-    'User already exists': 'El usuario ya existe',
+    'Confirmation Code': 'Codigo de confirmación',
+    'Create a new account': 'Crear una cuenta nueva',
+    Email: 'Email',
+    'Forgot Password': 'Se te olvidó la contraseña?',
     'Incorrect username or password':
       'Nombre de usuario o contraseña incorrecta',
     'Invalid password format': 'Formato de contraseña inválido',
     'Invalid phone number format':
       'Formato de n\xFAmero de tel\xE9fono inv\xE1lido.\nUtilice el formato de n\xFAmero de tel\xE9fono +12345678900',
+    'Loading...': 'Cargando...',
+    'New Password': 'Nueva contraseña',
+    Password: 'Contraseña',
+    'Phone Number': 'Número de teléfono',
+    'Resend a Code': 'Reenviar un código',
+    'Resend Code': 'Mandar codigo otra vez',
+    'Send Code': 'Enviar código',
+    'Sign In': 'Iniciar sesíon',
+    'Sign in to your account': 'Iniciar sesíon',
+    'Sign Out': 'Desconectar',
+    'Sign Up': 'Regístrase',
+    Skip: 'Omitir',
+    Submit: 'Enviar',
```

Comment:
What do you think about `consistent` for prettier?
https://prettier.io/docs/en/options.html#quote-props

Reply:
Yeah, I like “consistent.” Let’s update our prettier to use that. I’ll make that change in a coming PR. Good suggestion!